src/pages/patientView/SampleManager.tsx

```diff
@@ -159,7 +159,9 @@ class SampleManager {
     getComponentForSample(sampleId: string,
                           fillOpacity: number = 1,
                           extraTooltipText: string = '',
-                          additionalContent: JSX.Element|null = null)
+                          additionalContent: JSX.Element|null = null,
+                          onSelectGenePanel?:(name:string)=>void,
+                          tooltipIsDisabled?:boolean)
```

Comment:
this is subtle but I think it should be `disableTooltip`. `tooltipIsDisabled` sounds a bit off to me, almost like its describing the state of some other tooltip, whereas what its really doing is specifying whether the tooltip here should be disabled.